specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -1906,6 +2226,347 @@
           "description": "A textual explanation that can be used for display purposes to summarize the index value and category. For example, when the index value for Flight Delays is very good, the description will be \"Conditions are excellent for flying!\"."
         }
       }
+    },
+    "ActiveStormResponse": {
+      "type": "object",
+      "description": "all government-issued active storms",
```

Comment:
Change: 
all government-issued active storms

To:
All government-issued active storms.

Reply:
Updated